detector/aws_instance_invalid_type.go

```diff
@@ -6,7 +6,15 @@ import (
 	"github.com/wata727/tflint/issue"
 )
 
-func (d *Detector) DetectAwsInstanceInvalidType(issues *[]*issue.Issue) {
+type AwsInstanceInvalidTypeDetector struct {
```

Comment:
__[golint]__
* [golint] exported type AwsInstanceInvalidTypeDetector should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->